Client/src/Pages/Login/index.css

```diff
@@ -41,13 +41,13 @@
 
 #login-email-input {
   width: 308px;
-  height: 11px;
+  height: 1px;
   font-size: var(--env-var-font-size-medium);
 }
 
 #login-password-input {
   width: 308px;
-  height: 11px;
+  height: 1px;
   font-size: var(--env-var-font-size-medium);
 }
```

Comment:
Try to keep your pull requests modular, if you are working on a feature on the backend please make sure that all commits in the PR are related to that feature.

If we need to revert this PR because of an issue on the backend we will now also be reverting the front end, and this could be something we easily miss.

Best thing to do would be to remove this from your commit and open up another PR to fix this issue.

Reply:
Yeah, exactly.
It was just a quick fix that had to be done before being forgotten 😊

Reply:
Ya, it's a small fix, can you please move it to its own pull request so we can keep the PR related entirely to the Delete User feature in case we need to revert it?

Thanks!

Reply:
Ya, it's a small fix, can you please move it to its own pull request so we can keep this PR related entirely to the Delete User feature in case we need to revert it?

Thanks!

Reply:
Done for both